internal/play/play.go

```diff
@@ -50,6 +50,9 @@ func Play(api internal.APIInterface, query string) (string, error) {
 		}
 
 		err = api.Play(track.URI)
+		if err != nil {
+			return "", err
+		}
```

Comment:
This actually gets caught a few lines down... this must be a false positive. If `staticcheck` wants us to do this, let's add another check in the `else` block, and delete the check below the entire `if`/`else`

Reply:
Add another check in the else block and delete the check below the entire if/else because otherwise, staticcheck reports an error.

Reply:
Awesome, LGTM